File: Core/src/Internal/OftProtocolVersion.cs
namespace OpenFrameTransport.Internal;

/// <summary>
/// The OFT protocol version spoken by this implementation, sent as <see cref="Hail.Version"/> in
/// this side's hail (see Docs/OFT.md §3).
/// </summary>
internal static class OftProtocolVersion
{
    /// <summary>
    /// The current OFT protocol version string.
    /// </summary>
    public const string Current = "oft/1";
}

File: Ports/Java/src/main/java/org/blueheighliner/openframetransport/OftProtocolVersion.java
package org.blueheighliner.openframetransport;

/**
 * The OFT protocol version spoken by this implementation, sent in this side's hail (see
 * README.md &sect;3).
 */
final class OftProtocolVersion {
    /** The current OFT protocol version string. */
    static final String CURRENT = "oft/1";

    private OftProtocolVersion() {
    }
}
